i18n/2017/categories/misc/zh.md

```diff
@@ -0,0 +1 @@
+TODO: Need translation.
```

Comment:
If @michaelrambeau is k with the category message, i would need some help for the translation @frankyxhl  ;)

Reply:
Of course, I will help with Chinese translation. 😄